pycheribuild/config/compilation_targets.py

```diff
@@ -557,6 +557,51 @@ def _get_rootfs_project(self, xtarget: "CrossCompileTarget") -> "Project":
         return BuildCHERIBSD.get_instance(self.project, cross_target=xtarget)
 
 
+class CheriBSDMorelloTargetInfo(CheriBSDTargetInfo):
+    shortname = "CheriBSD-Morello"
+
+    def _get_sdk_root_dir_lazy(self):
+        return self.config.morello_sdk_dir
+
+    def get_cheribsd_sysroot_path(self) -> Path:
+        if self.target.is_aarch64(include_purecap=True):
+            return self._sysroot_path(self.sdk_root_dir, purecap_prefix="-morello-purecap",
+                                      hybrid_prefix="-morello-hybrid", nocheri_name="-aarch64")
+        return super().get_cheribsd_sysroot_path()
+
+    @classmethod
+    def triple_for_target(cls, target: "CrossCompileTarget", config: "CheriConfig", include_version):
+        if target.is_hybrid_or_purecap_cheri():
+            assert target.is_aarch64(include_purecap=True), "Only AArch64 purecap is supported with Morello toolchain"
```

Comment:
```suggestion
            assert target.is_aarch64(include_purecap=True), "AArch64 is the only CHERI target supported with the Morello toolchain"
```
but that also gets me thinking, do the CHERI-MIPS and CHERI-RISC-V tests all pass with the Morello toolchain? Bar being outdated, does it work for building CheriBSD etc for either of them? Because they really should, and if not there's something problematic going on with the Morello patches.

Reply:
Hopefully but I don't know if anyone has tried it.

Reply:
I would hope that they have at least run the unit tests and don't build only the AArch64 backend, but yeah I doubt anyone's tried anything more than that.